src/System.CommandLine.Benchmarks/DragonFruit/Perf_CommandLine_Help.cs

```diff
@@ -0,0 +1,57 @@
+﻿// Copyright (c) .NET Foundation and contributors. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System.CommandLine.Benchmarks.Helpers;
+using System.IO;
+using System.Linq;
+using System.Reflection;
+using System.Threading.Tasks;
+using BenchmarkDotNet.Attributes;
+
+namespace System.CommandLine.Benchmarks.DragonFruit
+{
+    /// <summary>
+    /// This is an end-to-end benchmark that measures the performance of --help option.
+    /// </summary>
+    [BenchmarkCategory(Categories.DragonFruit)]
+    [InvocationCount(3000)]
+    public class Perf_CommandLine_Help
```

Comment:
how does this differ from passing `--help` in `Perf_CommandLine_Help.cs`? Can we remove this or remove the call there?

Reply:
Removed duplicate `--help` test from `Perf_CommandLine_EntryPoint.cs`.